crates/wallet/src/wallet/mod.rs

```diff
@@ -1037,13 +1041,10 @@ impl Wallet {
     /// ```
     ///
     /// [`Anchor`]: bdk_chain::Anchor
-    pub fn get_tx(
-        &self,
-        txid: Txid,
-    ) -> Option<CanonicalTx<'_, Arc<Transaction>, ConfirmationTimeHeightAnchor>> {
+    pub fn get_tx(&self, txid: Txid) -> Option<WalletTx> {
```

Comment:
@notmandatory I think the documentation needs an update now, referring to `WalletTx` instead, wdyt ?

Reply:
Good catch, fixed.